src/renderer/components/swap/Swap.tsx

```diff
@@ -146,7 +145,7 @@ export const Swap = ({
 }: SwapProps) => {
   const intl = useIntl()
 
-  const unlockedWallet = useMemo(() => isLocked(keystore) || !hasImportedKeystore(keystore), [keystore])
+  const unlockedWallet = useMemo(() => !isLocked(keystore) && hasImportedKeystore(keystore), [keystore])
```

Comment:
Why to reverse the logic and not just rename this function to `lockedWallet`? _I know it's still WIP..._

Reply:
Yeah, that would actually be good too, will change it back.